databroker/_core.py

```diff
@@ -1105,6 +1105,13 @@ def fetch_external(self, start, stop):
         return {k: func(start, stop) for
                 k, func in self.external_fetchers.items()}
 
+    @property
+    def ES(self):
```

Comment:
Any better ideas for how to name this? We have `event_sources`, a list of event sources, and we have this -- a dict mapping the event source _name_ to the event source. Not sold on `ES`. attn @NSLS-II/dama

Reply:
`event_source_by_name`?

`sources_by_name`

`named_sources`

`named_es`

`es_by_name`

`names`